modules/README.md

```diff
@@ -120,6 +120,9 @@ Modules that are still in development and less stable are labeled with the
 
 * **[startup-script]** ![core-badge] : Creates a customizable startup script
   that can be fed into compute VMs.
+* **[htcondor-install]** ![community-badge] ![experimental-badge] : Creates
```

Comment:
nit: this is the only item in the module list that wraps to a second line. Would it be possible to wordsmith to reduce the length to fit on one line.

As viewed [here](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/00434b5f4477d6324152d18cd0e602982c7ccc3d/modules/README.md).

Reply:
Addressed